webpack.release.config.js

```diff
@@ -19,27 +19,9 @@ var plugins = require('./buildtools/webpack/plugins');
 var useRuntime = require('./buildtools/webpack/runtime');
 var usePolyfill = require('./buildtools/webpack/polyfill');
 
-// 1. entryConfig (node module main entry. minified, no polyfill, external dependencies)
+// 1. entryConfig (node module main entry. minified, no polyfill)
 var entryConfig = config('okta-sign-in.entry.js');
 entryConfig.output.filename = 'okta-sign-in.entry.js';
-entryConfig.externals = {
```

Comment:
does this imply `entry.js` will bundle all those dependencies (handlebar, underscore etc)?

Reply:
yes

Reply:
- why would do that.? and `entry.js` becomes same to `noPolyfillConfig`

Reply:
More information about why we want this change is here: https://oktainc.atlassian.net/browse/OKTA-320626

I think we probably can remove the no-polyfill entry and suggest people use entry.js from CDN for the no-polyfill option.
Moving deps to devDeps would reduce install time for the SIW, I'm OK with doing that. Sometimes people leave deps that are bundled in webpack for other reasons (like determining whether a security vuln is relevant or not). I'm not sure if we have a clear standard on that.

Reply:
Im not sure either and doesn't seem matter atm. most vuln concern we got so far is jQuery. also we shall try to keep 3rd-party as latest as possible